styles.css

```diff
@@ -0,0 +1,119 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  font-weight: 400;
+  background-color: #e5e5e5;
+}
+
+.header {
+  height: 80px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-color: #fff;
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: 0 auto;
+  background-color: white;
+}
+
+/* Hero section */
+.section-hero {
+  background-image: url("./image/header-shapes\ mobile.svg");
+  background-repeat: no-repeat;
+  padding: 100px 0;
+  background-color: white;
+  border-bottom-left-radius: 8rem;
```

Comment:
- [ ] [OPTIONAL] You should try to reduce the border radius to 4 rem. It looks close to the actual Figma design.